src/main/java/step1/qna/domain/Answer.java

```diff
@@ -1,12 +1,14 @@
-package qna.domain;
+package step1.qna.domain;
 
-import qna.NotFoundException;
-import qna.UnAuthorizedException;
+import step1.qna.CannotDeleteException;
+import step1.qna.NotFoundException;
+import step1.qna.UnAuthorizedException;
 
 import javax.persistence.*;
 
 @Entity
 public class Answer extends AbstractEntity {
+    private static final String EXCEPTION_MSG = "다른 사람이 쓴 답변이 있어 삭제할 수 없습니다.";
     @ManyToOne(optional = false)
```

Comment:
```suggestion
    private static final String EXCEPTION_MSG = "다른 사람이 쓴 답변이 있어 삭제할 수 없습니다.";

    @ManyToOne(optional = false)
```

상수와 인스턴스 변수 등 성격이 다른 것들 사이에는 빈 줄을 두어 가독성을 향상시켜 보아요!